src-ghc/Pact/Server/Server.hs

```diff
@@ -51,7 +52,10 @@ data Config = Config {
   _verbose :: Bool,
   _entity :: Maybe EntityName,
   _gasLimit :: Maybe Int,
-  _gasRate :: Maybe Int
+  _gasRate :: Maybe Int,
```

Comment:
Don't add anything else to this config, we'll use a repl function for public stuff

Reply:
Removed